lib/ABI/FunctionType.cpp

```diff
@@ -278,6 +285,36 @@ class ConvertionHelper {
     return Result;
   }
 
+  static InternalLayout getLayout(const model::CABIFunctionType &Function) {
+    InternalLayout Result;
+
+    size_t CurrentOffset = 0;
+    DistributedArguments Arguments = distributeArguments(Function.Arguments);
+    revng_assert(Arguments.size() == Function.Arguments.size());
+    for (size_t Index = 0; Index < Arguments.size(); ++Index) {
+      auto &Current = Result.Arguments.emplace_back();
+      Current.Registers = std::move(Arguments[Index].Registers);
+      if (Arguments[Index].SizeOnStack != 0) {
+        /// \todo maybe some kind of alignment considerations are needed here.
```

Comment:
We do not use doxygen within function bodies

```suggestion
        // TODO: maybe some kind of alignment considerations are needed here.
```

Reply:
Gone: https://github.com/revng/revng/pull/131/commits/505909f2fca23b0a400bb6c17031fac8481b8472.